packages/sitecore-jss-nextjs/package.json

```diff
@@ -39,6 +39,7 @@
     "@types/prop-types": "^15.7.4",
     "@types/react": "^17.0.15",
     "@types/react-dom": "^17.0.9",
+    "@types/sinon": "^7.5.0",
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.3",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
```

Comment:
Need to remove "chai-spies" and "@types/chai-spies"?

Reply:
Looks like these are still here. These should be removed (since we're now using sinon for spies), right?

Reply:
removed now.